### DISCUSSION TITLE
Git is not working in when I open workspace in theia

### DISCUSSION THREAD

**Discussion by balakrishna-maduru** [eclipse-theia/theia#13571] [General]
When i upgrade theia to 1.47.1 workspace with multiple repositories and sinle repo started not showing the repository selection and branch in status bar in theia below image is for referance.
![image](https://github.com/eclipse-theia/theia/assets/64999051/677d2475-655e-44c6-b1fa-a088f78dd0e8)
@theia/bulk-edit 1.45.1
@theia/callhierarchy 1.45.1
@theia/console 1.45.1
@theia/core 1.45.1
@theia/debug 1.45.1
@theia/editor 1.45.1
@theia/editor-preview 1.45.1
@theia/file-search 1.45.1
@theia/filesystem 1.45.1
@theia/git 1.45.1
@theia/markers 1.45.1
@theia/messages 1.45.1
@theia/mini-browser 1.45.1
@theia/monaco 1.45.1
@theia/navigator 1.45.1
@theia/notebook 1.45.1
@theia/outline-view 1.45.1
@theia/output 1.45.1
@theia/plugin-ext 1.45.1
@theia/plugin-ext-vscode 1.45.1
@theia/preferences 1.45.1
@theia/preview 1.45.1
@theia/process 1.45.1
@theia/scm 1.45.1
@theia/scm-extra 1.45.1
@theia/search-in-workspace 1.45.1
@theia/task 1.45.1
@theia/terminal 1.45.1
@theia/test 1.45.1
@theia/timeline 1.45.1
@theia/typehierarchy 1.45.1
@theia/userstorage 1.45.1
@theia/variable-resolver 1.45.1
@theia/workspace 1.45.1

**Comment by JonasHelming** [comment]
Hi,

you list 1.45.1 extensions below the screenshot, what is the meaning of this? Also, please be aware that the Theia Git extension is deprecated, we recommend using the VS Code built-in Git support instead.

Best regards,
Jonas
